cabal-install/Distribution/Client/ProjectOrchestration.hs

```diff
@@ -483,6 +484,14 @@ printPlan verbosity
     showFlagValue (f, False)  = '-' : showFlagName f
     showFlagName (PD.FlagName f) = f
 
+    showProfiling elab =
+        (if elabProfLib elab
+            then " profile"
+            else "") ++
+        (if elabProfExe elab
+            then " exe-profile"
+            else "")
```

Comment:
Yes it's bad.

Here's another thought; what if the output log message output the flags that we would pass to the configure script (maybe not all of them, but some of the important ones)? That would make things clearer.